.github/workflows/release.yml

```diff
@@ -48,23 +47,13 @@ jobs:
     needs: [test]
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
+        os: [ubuntu-latest]
         include:
           - os: ubuntu-latest
             rust: nightly
             target: x86_64-unknown-linux-musl
             bin: kct
             name: kct-linux-amd64
-          - os: windows-latest
-            rust: nightly
-            target: x86_64-pc-windows-msvc
-            bin: kct.exe
-            name: kct-windows-amd64.exe
-          - os: macOS-latest
-            rust: nightly
-            target: x86_64-apple-darwin
-            bin: kct
-            name: kct-darwin-amd64
     runs-on: ${{ matrix.os }}
```

Comment:
You don't need to use a matrix strategy here, you just have one `os`!
And you can set `rust`, `target`, `bin` and `name` as environment variable to use inside your workflow.

Reply:
I 100% agree and have kept it purely out of laziness with the hopes of building for multiple platforms someday.

I will remove it.

Reply:
Well, you don't need to use the `workflow` commands here, you just use them if you want to get an output of any step to use in another step!

You can set env variables for all steps, see the documentation [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idenv)

Reply:
I wasn't aware of those. Thanks for bringing it in.